examples/artifacts/plot_detect_riemannian_potato_field_EEG.py

```diff
@@ -0,0 +1,280 @@
+"""
+===============================================================================
+Online Artifact Detection with Riemannian Potato Field
+===============================================================================
+
+Example of Riemannian Potato Field [1]_ applied on EEG time-series to detect
```

Comment:
```suggestion
Example of Riemannian Potato Field (RPF) [1]_ applied on EEG time-series to detect
```
as you use the acronym below